recipes/multidimio/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set name = "multidimio" %}
+{% set version = "0.1.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/multidimio-{{ version }}.tar.gz
+  sha256: 45bb0f2070d39364e3a22eab0a6b42dc58adfeaf775e21ffde5fc04a203263b2
+
+build:
+  entry_points:
+    - mdio = mdio.__main__:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8,<3.11
+    - pip
+    - poetry

```

```suggestion
    - poetry-core
```

Ref: https://github.com/TGSAI/mdio-python/blob/429a6cbf7dcfa5037f0491952e150a532ca9b8ea/pyproject.toml#L109